### ISSUE TITLE
Prompt is empty - please help

### ISSUE THREAD

**Issue by simonweil** [bhilburn/powerlevel9k#36]
I'm having trouble getting powerlevel9k to work for me, this is the output:
![screen shot 2015-06-08 at 12 57 16 am](https://cloud.githubusercontent.com/assets/2317562/8026306/4f3bb6f6-0d79-11e5-8bd2-a3dff41c4aef.png)

My .zshrc is:
```zsh
#!/usr/bin/env zsh

source ~/.dotfiles/non-packaged-repos/antigen/antigen.zsh

antigen use oh-my-zsh

export DEFAULT_USER=simonweil
POWERLEVEL9K_LEFT_PROMPT_ELEMENTS=(context dir vcs status)
POWERLEVEL9K_RIGHT_PROMPT_ELEMENTS=(rbenv virtualenv history time)
antigen theme bhilburn/powerlevel9k powerlevel9k

# Tell antigen that you're done.
antigen apply
```

When I use `antigen theme funky` all is fine.

As I'm new to zsh I have no real idea what to do...

**Comment by dritter** [comment]
Hi @simonweil !

Thanks for the report! I am currently working on support for `prezto` and `antigen`. This is in my fork https://github.com/dritter/powerlevel9k/tree/dritter/prezto, but it is not ready yet. Hopefully, I am getting somewhere, before going on vacation. Stay tuned.

As a workaround: The Theme is tested with `oh-my-zsh`. It should work if you are willing to use that temporarily..

**Comment by simonweil** [comment]
Oh, I see.
I'll wait for full support, but I'll be glad to help out testing or in any other way you think I can give a hand.

**Comment by bh** [comment]
@simonweil 
``
antigen theme dritter/powerlevel9k powerlevel9k --branch=dritter/prezto
``
Until it is merged to upstream :-)

**Comment by brcolow** [comment]
This worked for me with Antigen! Thank you so much, my terminal looks *beautiful*

**Comment by simonweil** [comment]
@bh - Works great, thank you!!!

**Comment by simonweil** [comment]
One thing doesn't work well.
When I customize the prompt elements,
the "status" part shows only an X when the command failed but nothing otherwise.
When using the defaults, all is fine.

Also the virtualenv doesn't seem to work.

**Comment by bhilburn** [comment]
Hi all -

I am working on pulling this branch upstream in the next couple of days. Keep an eye on Pull Request #38!

**Comment by dritter** [comment]
@simonweil could you open separate issues for the broken features? I'll have an eye on that in a few weeks.. Thx.

**Comment by bhilburn** [comment]
Closed with merge from pull request #38, antigen support confirmed in bug #40!
